docs/deployment/ecs.mdx

```diff
@@ -0,0 +1,135 @@
+                    - DATABASE_CONNECTION_STRING
+                    - AUTH_TYPE
+                    - KEEP_JWT_SECRET
+                    - KEEP_DEFAULT_USERNAME
+                    - KEEP_DEFAULT_PASSWORD
+                    - SECRET_MANAGER_TYPE
+                    - SECRET_MANAGER_DIRECTORY
+                    - USE_NGROK            
+                    - KEEP_API_URL           
+                    - PUSHER_APP_ID        
+                    - PUSHER_APP_KEY         
+                    - PUSHER_APP_SECRET     
+                    - PUSHER_HOST     
+                    - PUSHER_PORT 
+            ```
+        - Review and create your task definition.
+
+    ### Task Definition 2 (Backend - keepAPI):
+
+        - Configure your container definitions as below:
+            ```
+            - Infrastructure Requirements:
+                - Launch Type: AWS Fargate
+                - OS, Architecture, Network mode: Linux/X86_64
+                - Task Size: 
+                    - CPU: 0.5 vCPU
+                    - Memory: 4 GB
+                - Task Role and Task Execution Role are optional if yoy plan on using secrets manager for example then create a task execution role to allow access to the secret manager you created.
+            - Container Details:
+                - Name: keep-backend
+                - Image URI: us-central1-docker.pkg.dev/keephq/keep/keep-api:latest
+                - Ports Mapping:
+                    - Container Port: 8080
+                    - Protocol: TCP
+                - Environment Variables: (This can be static of you can use parameter store or secrets manager)
+                    - DATABASE_CONNECTION_STRING
+                    - AUTH_TYPE
+                    - KEEP_JWT_SECRET
+                    - KEEP_DEFAULT_USERNAME
+                    - KEEP_DEFAULT_PASSWORD
+                    - SECRET_MANAGER_TYPE
+                    - SECRET_MANAGER_DIRECTORY
+                    - USE_NGROK            
+                    - KEEP_API_URL           
+                    - PUSHER_APP_ID        
+                    - PUSHER_APP_KEY         
+                    - PUSHER_APP_SECRET     
+                    - PUSHER_HOST     
+                    - PUSHER_PORT 
+            - Storage:
+                - Volume Name: keep-efs
+                - Configuration Type: Configure at task definition creation
+                - Volume type: EFS
+                - Storage configurations:
+                    - File system ID: Select an exisiting EFS filesystem or create a new one
+                    - Root Directory: /
+                - Container mount points: 
+                    - Container: select the container you just created
+                    - Source volume: keep-efs
+                    - Container path: /app
+                    - Make sure that Readonly is not selected
+            ```
+        - Review and create your task definition.
+
+    ### Task Definition 3 (Websocket):
```

Comment:
FYI it's worth mentioning that this might be redundant if one chooses to use [Pusher](https://pusher.com/), a "managed" version of Soketi (kinda)

Reply:
just makes the deployment a bit easier

Reply:
Ok sure will update it.